examples/utils.py

```diff
@@ -9,3 +16,163 @@ def build_parser(description: str):
         "--debug", dest="log_level", action="store_const", const=logging.DEBUG, default=logging.INFO
     )
     return parser
+
+
+class SummaryLogger:
```

Comment:
@azawlocki hmm... question - is this class indeed only usable by the example scripts? sure, it's only used by the examples because we don't have any other code that could use this functionality now - but maybe third-party requestor agents could also benefit from having access to this kind of summarized logger? 

in this case - maybe `SummaryLogger` should also be made part of `yapapi.log` module?

Reply:
I can't make up my mind about this.

On the one hand, I like the idea of keeping the core library small and only including the most basic "event handlers" (that would be the functions `log_event` and `log_event_json` in `yapapi.log`) and let users of the library implement their own ways of handling and aggregating computation events, in the form of GUIs, web services, etc. (the `SummaryLogger` would be a very simple example of this category).

On the other hand, maybe the `SummaryLogger` is useful and general enough to quality for inclusion in `yapapi` together with the basic loggers? If we put it in `examples` we will have to maintain it anyway and there are already problems with importing it into `blender.py` from the parent dir.

Reply:
Moved `SummaryLogger` to `yapapi.log` in https://github.com/golemfactory/yapapi/pull/41/commits/ae4461c0f5a11e23a11d36c249eef9d0a42b7969